codegen/arrai/grpc_client.arrai

```diff
@@ -56,7 +56,7 @@
             	"context"
             	"time"
 
-            	pb "${proj_location}/${jsontool.attr("go_pb_package", app)}"
+            	pb "${proj_location}/${//str.sub(jsontool.attr("package", app), ".", "/")}"
```

Comment:
sub is in //seq not //str now: https://github.com/arr-ai/arrai/pull/240

```suggestion
            	pb "${proj_location}/${//seq.sub(jsontool.attr("package", app), ".", "/")}"
```

Reply:
Ack